### ISSUE TITLE
Play2 example not compatible with Play 2.4+

### ISSUE THREAD

**Issue by blast-hardcheese** [kouphax/sbt-tasks#9]
It seems like Play 2.4 has dropped support for `StaticApplication` ([commit](https://github.com/playframework/playframework/commit/688f651064fff2f2189d9e9c0734e04b6d4c304d)), which the example in `README.md` uses. According to the 2.5.x docs for "[Launch the Interactive Console](https://www.playframework.com/documentation/2.5.x/PlayConsole#Launch-the-interactive-console)", a reliable method for starting an application seems to be:

    import play.api._
    val env = Environment(new java.io.File("."), this.getClass.getClassLoader, Mode.Dev)
    val context = ApplicationLoader.createContext(env)
    val loader = ApplicationLoader(context)
    val app = loader.load(context)
    Play.start(app)
    import Play.current 

(which seems to work for me, at least, despite being a bit wordy.)

A good definition of `PlayTask` seems to be:

```
package tasks

import play.api._
import Play.current

trait PlayTask extends Runnable {
  val env = Environment(new java.io.File("."), this.getClass.getClassLoader, Mode.Dev)
  val context = ApplicationLoader.createContext(env)
  val loader = ApplicationLoader(context)
  val app = loader.load(context)

  def work(): Unit

  def run(): Unit = {
    Play.start(app)
    work()
    Play.stop(app)
  }
}
```

**Comment by kouphax** [comment]
Thanks for this. I'm happy for you to open a PR to include this example in the README.md.  otherwise let me know and I'll do it myself.

I'd like to keep both example in the README and mark them with Play versions.

**Comment by blast-hardcheese** [comment]
@kouphax PR opened-- please let me know if there are styling tweaks you'd like

**Comment by blast-hardcheese** [comment]
PR merged, closing
